docs/gatsby-config.js

```diff
@@ -20,7 +20,22 @@ module.exports = {
       options: {
         defaultLayouts: {
           default: require.resolve('./src/components/layouts/default.tsx')
-        }
+        },
+        gatsbyRemarkPlugins: [
+          {
+            resolve: 'gatsby-remark-inline-codesandbox',
```

Comment:
i'm finding this plugin works but doesn't give us a lot of flexibility. let's try it out but i have a backup plan